module/VuFind/tests/unit-tests/src/VuFindTest/Service/CurrencyFormatterTest.php

```diff
@@ -43,59 +43,26 @@
  */
 class CurrencyFormatterTest extends \PHPUnit\Framework\TestCase
 {
-    /**
-     * Locale (for restoration after testing)
-     *
-     * @var string
-     */
-    protected $locale;
-
-    /**
-     * Standard setup method
-     *
-     * @return void
-     */
-    public function setUp(): void
-    {
-        // store current default and set a value for consistency in testing
-        $this->locale = setlocale(LC_MONETARY, 0);
-        $locales = ['en_US.UTF8', 'en_US.UTF-8', 'en_US'];
-        if (false === setlocale(LC_MONETARY, $locales)) {
-            $this->markTestSkipped('Problem setting up locale');
-        }
-    }
-
-    /**
-     * Standard teardown method
-     *
-     * @return void
-     */
-    public function tearDown(): void
-    {
-        // restore current default
-        setlocale(LC_MONETARY, $this->locale);
-    }
-
     /**
      * Test the class
      *
      * @return void
      */
     public function testFormatting()
     {
-        // test default settings
-        $cc = new \VuFind\Service\CurrencyFormatter();
-        $this->assertEquals('$3,000.00', $cc->convertToDisplayFormat(3000));
-        $this->assertEquals('€3,000.00', $cc->convertToDisplayFormat(3000, 'EUR'));
-
-        // test override default currency
-        $cc = new \VuFind\Service\CurrencyFormatter('EUR');
+        // test overriding default currency
+        $cc = new \VuFind\Service\CurrencyFormatter('EUR', 'en_US');
```

Comment:
Since this test is now explicitly setting the currency and the locale, I don't see how it's different from the "test overriding both" below. Does it serve any purpose?

Could we still test the default case by using the NumberFormatter directly in the test to generate the expected value? I'm not sure if that really tells us anything, but maybe it helps exercise more code paths...

Reply:
I restored the default value tests. They feel quite artificial, but perhaps there's some value. :)